concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-168	168	granada	pomegranate			Noun	5.98	7	101	
+Desrochers-2010-330-169	169	hielo	ice	ICE	617	Noun	6.17647058823529	5	173	6.76
+Desrochers-2010-330-170	170	hierba	grass	GRASS	606	Noun	5.99019607843137	6	132	6.31
+Desrochers-2010-330-171	171	incendio	fire	FIRE	221	Noun	5.64705882352941	8	83	4.93
+Desrochers-2010-330-172	172	joya	jewel	JEWEL	1029	Noun	5.56862745098039	4	26	6.05
+Desrochers-2010-330-173	173	kilómetro	kilometer			Noun	6.38235294117647	9	45	6.36
+Desrochers-2010-330-174	174	labio	lip	LIP	478	Noun	6.46078431372549	5	45	6.24
+Desrochers-2010-330-175	175	lápiz	pencil	PENCIL	2578	Noun	6.53921568627451	5	39	6.16
+Desrochers-2010-330-176	176	lata	can	CAN	1051	Noun	6.24509803921569	4	58	6.06
+Desrochers-2010-330-177	177	levita	overskirt			Noun	2.96078431372549	6	8	
+Desrochers-2010-330-178	178	libro	book	BOOK	963	Noun	6.67647058823529	5	1088	6.84
+Desrochers-2010-330-179	179	línea	line	LINE	1048	Noun	5.99019607843137	5	488	6.15
+Desrochers-2010-330-180	180	llama	flame	FLAME	889	Noun	5.05882352941176	5	631	4.95
+Desrochers-2010-330-181	181	llave	key	KEY	1472	Noun	6.63725490196079	5	128	6.71
+Desrochers-2010-330-182	182	mañana	tomorrow	TOMORROW	1329	Noun	6.69607843137255	6	1429	7
+Desrochers-2010-330-183	183	mancha	stain			Noun	6.05882352941176	6	91	6.56
+Desrochers-2010-330-184	184	mano	hand	HAND	1277	Noun	6.57843137254902	4	2168	6.53
+Desrochers-2010-330-185	185	mantel	tablecloth			Noun	5.84313725490196	6	33	6.27
+Desrochers-2010-330-186	186	marciano	martian			Noun	4.34313725490196	8	8	
+Desrochers-2010-330-187	187	miedo	fear	FEAR (FRIGHT)	781	Noun	6.16666666666667	5	839	6.19
+Desrochers-2010-330-188	188	moneda	currency			Noun	6.57843137254902	6	110	6.31
+Desrochers-2010-330-189	189	monstruo	monster			Noun	5.14705882352941	8	117	6.06
+Desrochers-2010-330-190	190	montaña	mountain	MOUNTAIN	639	Noun	6.08823529411765	7	193	6.38
+Desrochers-2010-330-191	191	mosca	fly	FLY (INSECT)	1504	Noun	6.01960784313726	5	60	5.94
+Desrochers-2010-330-192	192	mueble	furniture			Noun	6.17647058823529	6	42	6.27
+Desrochers-2010-330-193	193	mundo	world	WORLD	965	Noun	6.44117647058824	5	3783	6.44
+Desrochers-2010-330-194	194	nieve	snow	SNOW	784	Noun	5.82352941176471	5	220	5.41
+Desrochers-2010-330-195	195	nuez	nut	NUT	191	Noun	5.42156862745098	4	24	5.17
+Desrochers-2010-330-196	196	ola	wave	WAVE	978	Noun	5.5	3	62	6.76
+Desrochers-2010-330-197	197	página	page			Noun	6.52941176470588	6	178	6.64
+Desrochers-2010-330-198	198	pájaro	bird	BIRD	937	Noun	6.29411764705882	6	116	6.5
+Desrochers-2010-330-199	199	papel	paper	PAPER	648	Noun	6.66666666666667	5	1028	6.66
+Desrochers-2010-330-200	200	paso	step	STEP	1158	Noun	5.88235294117647	4	1331	6.24
+Desrochers-2010-330-201	201	pata	paw	PAW	2965	Noun	5.92156862745098	4	78	6.45
+Desrochers-2010-330-202	202	patata	potato	POTATO	593	Noun	6.51960784313726	6	33	6.75
+Desrochers-2010-330-203	203	pelota	ball	BALL	1845	Noun	6.40196078431373	6	103	6.49
+Desrochers-2010-330-204	204	perra	bitch			Noun	6.22	5	60	6.47
```

Comment:
she-dog, we have that in concepticon, maybe as "female dog". So bitch is right, but it means the original meaning.

Reply:
Yes, 2481 | FEMALE DOG (even though can be used as a derogatory term just as in English)